shell/format.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 base_dir=$(dirname $(dirname $0))
-targets="${base_dir}/*.py ${base_dir}/examples/ ${base_dir}/keras_nlp/"
+targets="${base_dir}/*.py ${base_dir}/examples/ ${base_dir}/keras_nlp/ ${base_dir}/tools/checkpoint_conversion/*.py"
```

Comment:
Ah I missed this on the other PR, but will this work if we just do `${base_dir}/tools/` here? I think black and isort must avoid formatting non-python files, as they don't format all the stuff in examples.

Reply:
If this works, we should also make the change this change to `lint.sh` too.